embed.fnc

```diff
@@ -3476,8 +3476,8 @@ Sxd	|SV*	|find_uninit_var|NULLOK const OP *const obase \
 		|NN const char **desc_p
 #endif
 
-Ap	|GV*	|gv_fetchpvn_flags|NN const char* name|STRLEN len|I32 flags|const svtype sv_type
-Ap	|GV*	|gv_fetchsv|NN SV *name|I32 flags|const svtype sv_type
+Adp	|GV*	|gv_fetchpvn_flags|NN const char* name|STRLEN len|I32 flags|const svtype sv_type
+Adp	|GV*	|gv_fetchsv|NN SV *name|I32 flags|const svtype sv_type
```

Comment:
the extra `p` is tagging the `Perl_` prefix for the functions
view https://github.com/Perl/perl5/blob/29a07d66c047f476aeaa1f95573fe7765939f158/embed.fnc#L474

Reply:
This comment is still unresolved.

Reply:
I don't understand this comment.  Both of these are implemented as functions with the Perl_ prefix.  I added a 'd' to indicate they are now commented.  But the p was already there.  And should be there

Reply:
@atoomic you approved this PR, but this comment is still unresolved. Could you resolve the comment or further explain what you meant by it?